app/src/main/java/live/hms/app2/model/TokenRequest.kt

```diff
@@ -4,6 +4,7 @@ import com.google.gson.annotations.SerializedName
 
 data class TokenRequest(
   @SerializedName("room_id") val roomId: String,
-  @SerializedName("user_id") val userId: String,
-  @SerializedName("role") val role: String
+  @SerializedName("user_name") val username: String,
```

Comment:
@adityaa30 Use `user_id` instead of `user_name`

Reply:
What's `user_id`? The app initially take input only `name` and `room-id`